app/lib/calculation_output.rb

```diff
@@ -0,0 +1,7 @@
+class CalculationOutput
+  def initialize(capital_collation:)
+    @capital_collation = capital_collation
```

Comment:
I don't suppose there's an easy way to specify type-checking for these properties?

Reply:
I'm interested in Sorbet. Let's discuss this outside this ticket